src/options.js

```diff
@@ -0,0 +1,159 @@
+  n: 'name',
+  v: 'version',
+  r: 'resource',
+  a: 'resource',
+  p: 'python',
+  f: 'flag',
+  c: 'configure',
+  m: 'make',
+  vc: 'vcBuild',
+  s: 'snapshot',
+  b: 'bundle',
+  cli: 'enableNodeCli',
+  h: 'help',
+  l: 'loglevel'
+}
+const argv = parseArgv(process.argv, { alias, default: defaults })
+const help = `
+nexe --help              CLI OPTIONS
+
+  -i   --input      =index.js               -- application entry point
+  -o   --output     =nexe.exe               -- path to output file
+  -t   --target     =win32-x64-6.10.3       -- *target a prebuilt binary
+  -n   --name       =nexe.js                -- file name for error reporting at run time
+  -v   --version    =${padRight(process.version.slice(1), 23)}-- node version
+  -p   --python     =/path/to/python2       -- python executable
+  -f   --flag       ="--expose-gc"          -- *v8 flags to include during compilation
+  -c   --configure  ="--with-dtrace"        -- *pass arguments to the configure command
+  -m   --make       ="--loglevel"           -- *pass arguments to the make command
+  -vc  --vcBuild    =x64                    -- *pass arguments to vcbuild.bat
+  -s   --snapshot   =/path/to/snapshot      -- build with warmup snapshot
+  -r   --resource   =./paths/**/*           -- *embed file bytes within the binary
+  -b   --bundle     =webpack-config.js      -- use default configuration or provide custom webpack config
+       --temp       =./path/to/temp         -- nexe temp files (for downloads and source builds)
+       --ico                                -- file name for alternate icon file (windows)
+       --rc-*                               -- populate rc file options (windows)
+       --clean                              -- force download of sources
+       --enableNodeCli                      -- enable node cli enforcement (blocks app cli)
+       --sourceUrl                          -- pass an alternate source (node.tar.gz) url
+       --silent                             -- disable logging
+       --verbose                            -- set logging to verbose
+
+       -* variable key name                 * option can be used more than once
+
+
+
+`.trim()
+
+function flattenFilter (...args) {
+  return [].concat(...args).filter(x => x)
+}
+
+/**
+ * Extract keys such as { "rc-CompanyName": "Node.js" } to
+ * { CompanyName: "Node.js" }
+ * @param {*} match
+ * @param {*} options
+ */
+function extractCliMap (match, options) {
+  return Object.keys(options).filter(x => match.test(x))
+    .reduce((map, option) => {
+      const key = option.split('-')[1]
+      map = map || {}
+      map[key] = options[option]
+      delete options[option]
+      return map
+    }, null) || {}
+}
+
+function tryResolveMainFileName () {
+  let filename = 'nexe'
+  try {
+    const file = require.resolve(process.cwd())
+    filename = basename(file).replace(extname(file), '')
+  } catch (_) {}
+
+  return filename === 'index' ? ('nexe_' + Date.now()) : filename
```

Comment:
Change to `let filename` and `!filename || filename === 'index' ? ...`.  The help default of `=nexe.js` will be off because it doesn't include the timestamp.